cli/cmd/component-delete.go

```diff
@@ -15,6 +15,7 @@
 package cmd
 
 import (
+	"context"
```

Comment:
I'd put this changes in the modules update commit. It will make it easier to bisect/revert then. What do you think? (something which should go to development guidelines I guess).

Reply:
I don't have a strong opinion here. I guess bisectability is nice, I'll change this.